### ISSUE TITLE
Clarify intented use

### ISSUE THREAD

**Issue by bdsl** [jolicode/ffi-uuid#7]
This looks like a nice library, but from the readme it isn't clear to me exactly what it's for. Specifically what advantages does this have over [ramsey/uuid](https://github.com/ramsey/uuid)?

Might be worth adding something to the readme to clarify. Is it purely to serve as demonstration of FFI? Does it have performance advantages? Is this just to give us an extra option?

**Comment by lyrixx** [comment]
Hello. I wrote [an article](https://jolicode.com/blog/php-7-4-et-ffi-ce-quil-faut-retenir) yesterday about that. But unfortunately, it's only in French for now. I'm sorry about that.

Anyway ramsey/uuid is an extra layer on top of the PECL (or random_bytes, or big number, it depends of what is available). So ramsey/uuid is out of context here. But at some point this package could be used by ramsey/uuid. And the PECL relies on libuuid.

This lib replace the PECL by binding PHP directly to libuuid via FFI. But it's slower than the PECL :grimacing: 

So basically, for now it's more a POC to demonstrate the power of FFI. But if the FFI implementation (in PHP Core) becomes faster, then it could be better at some point to use this package instead of the PECL

**Comment by bdsl** [comment]
Sounds good, thanks for clarifying - looking forward to using this in prod in a year or two when FFI is fast then. I can make a PR to add that to the readme if you like.

**Comment by lyrixx** [comment]
Oh' this is very nice. Thanks you
